examples/experimental-9p.yaml

```diff
@@ -0,0 +1,22 @@
+# This example requires Lima v0.7.0 or later.
+images:
+# Try to use release-yyyyMMdd image if available. Note that release-yyyyMMdd will be removed after several months.
+# ⚠️ release-20220309 is known to be broken on aarch64: https://github.com/lima-vm/lima/issues/712
```

Comment:
This warning can be now removed

Reply:
Done (rebased, and re-diffed with ubuntu)